public/components/common/search/search.tsx

```diff
@@ -255,19 +254,6 @@ export const Search = (props: any) => {
           </>
         )}
       </EuiFlexGroup>
-      {searchError && searchError.error && (
-        <EuiFlexGroup gutterSize="s" justifyContent="flexStart" alignItems="flexStart">
-          <EuiFlexItem>
-            <EuiCallOut
-              title={JSON.parse(searchError.message).error.reason}
-              color="danger"
-              iconType="alert"
-            >
-              <p>{JSON.parse(searchError.message).error.details}</p>
-            </EuiCallOut>
-          </EuiFlexItem>
-        </EuiFlexGroup>
-      )}
```

Comment:
Was this removed as it was duplicate? I remember we also se the error on the toast message.

Reply:
Need to work with UX on this for further iterations